department-of-veterans-affairs/caseflow
# Always create a job note for Inbox-worthy fails and successes

Connects #12926 

### Description
As per AMO's feedback on the async messaging demo, all Inbox message types should have an analogous job note, so that CAs can screenshot the job page for a complete history and not worry about excluding details from other jobs.

### Acceptance Criteria
- [ ] When a job first fails after 24 hours, a job note is created.
- [ ] When a job succeeds after 24 hours, a job note is created.

### Testing Plan
1. Create or find a pending HigherLevelReview and ensure its submission is at least 24 hours in the past (use any Postgres client)
2.
```
pry> hlr.update_error!("any error")
pry> hlr.processed!
```
3. Verify that `/asyncable_jobs/HigherLevelReview/jobs/<ID>` shows job messages for those two events.
4. Tests updated.

### User Facing Changes
Sample screenshot of the language in the new job notes.
<img width="553" alt="Screen Shot 2019-12-18 at 10 41 50" src="https://user-images.githubusercontent.com/282869/71100437-0665f900-2183-11ea-8e8c-463362cbf7ba.png">
